git-branchless/tests/command/test_navigation.rs

```diff
@@ -600,7 +600,7 @@ fn test_checkout_pty_branch() -> eyre::Result<()> {
         let (stdout, _stderr) = git.run(&["smartlog"])?;
         insta::assert_snapshot!(stdout, @r###"
         :
-        @ 62fc20d (master) create test1.txt
+        @ 62fc20d (> master) create test1.txt
```

Comment:
I was also hoping that `git next` and `git prev` would automatically check out the relevant branch when moving onto a certain commit. Do you know why they're not doing that? Maybe there's actually just no test for a `git next`/`git prev` which moves onto a commit with a branch pointing to it?

Reply:
They do automatically work! Looks like there are just no tests covering that case (I can add some if that's useful)

Here's an asciinema of it in action: 
[![asciicast](https://asciinema.org/a/M1emyaWr05JWRyK0gH5HrRIPG.svg)](https://asciinema.org/a/M1emyaWr05JWRyK0gH5HrRIPG)